examples/tensorflow-benchmarks/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM rongou/horovod
```

Comment:
Plug: there are now official Horovod docker images at https://hub.docker.com/r/uber/horovod/

Reply:
Switched to it. But the tensorflow/benchmarks head seems no longer compatible with tf1.8.0, so had to reset to a previous commit.